.travis.yml

```diff
@@ -1,146 +1,160 @@
-dist: xenial
-
-os: linux
+sudo: required
 
 language: php
 
 services:
-  - mysql
+    - mysql
 
 cache:
-  apt: true
-  directories:
-    - $HOME/.composer/cache
-    - $HOME/.npm
```

Comment:
@tn3rb in case we will go with yarn, we will need to decide if we want to keep both cache directories.